recipes/interpolation/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.8" %}
+{% set sha256 = "2debe8a5fc000668edbf1b16bdc8de6e8830047bc3f3ffcb2a5636c16137b004" %}
+
+package:
+  name: interpolation
+  version: {{ version }}
+
+source:
+  fn: interpolation.py-{{ version }}.tar.gz
+  url: https://github.com/EconForge/interpolation.py/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  skip: True  # [py>35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numba
+    - numpy
+    - pandas
+    - tempita
+    - scipy
+
+test:
+  imports:
+    - interpolation
+    - interpolation.smolyak
+    - interpolation.splines
+
+about:
+  home: https://github.com/econforge/interpolation.py
+  license: BSD-2
```

Comment:
We try to standardize the license as much as possible. So please change this to `BSD 2-Clause` (it can be done later in the feedstock).